metadata.json

```diff
@@ -56,7 +56,7 @@
     {"name":"zack/r10k",                      "version_requirement":">= 2.3.0"},
 
     {"name":"puppetlabs/acl",                 "version_requirement":">= 1.0.0"},
-    {"name":"puppetlabs/dsc",                 "version_requirement":">= 1.0.0"},
+    {"name":"puppetlabs/dsc",                 "version_requirement":"1.2.0"},
```

Comment:
I mean that since Josh pinned powershell last week, and you pinned this one this week, are they both pinned to the right versions?